projects/plugins/jetpack/modules/carousel/jetpack-carousel.js

```diff
@@ -1624,15 +1259,53 @@
 				settings.startIndex = 0; // -1 returned if can't find index, so start from beginning
 			}
 
-			domUtil.emitEvent( carousel.container, 'jp_carousel.beforeOpen' );
+			domUtil.emitEvent( carousel.overlay, 'jp_carousel.beforeOpen' );
+			carousel.gallery.innerHTML = '';
 
-			domUtil.fadeIn( carousel.container, function () {
-				domUtil.emitEvent( carousel.container, 'jp_carousel.afterOpen' );
-			} );
+			// Need to set the overlay manually to block or swiper does't initialise properly.
+			carousel.overlay.style.opacity = 0;
```

Comment:
Should this be `carousel.overlay.style.opacity = 1;` ? 

Otherwise the black background doesn't show.

Reply:
It is set to 1 by the fadeIn method further down.